src/vue/backend/.npmrc

```diff
@@ -0,0 +1,2 @@
+@fortawesome:registry=https://npm.fontawesome.com/
+//npm.fontawesome.com/:_authToken=49A3C68C-3FE5-4C56-9DCE-85E4DDBE3BBA
```

Comment:
Since this is a public repo, I don't think we want our auth token available

Reply:
It should be `//npm.fontawesome.com/:_authToken=$TOKEN` and the token set in our local .env file which shouldn't be committed. We'll need to generate a new auth token for font awesome now to.

Reply:
agreed